Mathlib/Algebra/Category/BialgebraCat/Basic.lean

```diff
@@ -0,0 +1,199 @@
+/-
+Copyright (c) 2024 Amelia Livingston. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Amelia Livingston
+-/
+import Mathlib.Algebra.Category.CoalgebraCat.Basic
+import Mathlib.Algebra.Category.AlgebraCat.Basic
+import Mathlib.RingTheory.Bialgebra.Equiv
+
+/-!
+# The category of bialgebras over a commutative ring
+
+We introduce the bundled category `BialgebraCat` of bialgebras over a fixed commutative ring `R`
+along with the forgetful functors to `CoalgebraCat` and `AlgebraCat`.
+
+This file mimics `Mathlib.LinearAlgebra.QuadraticForm.QuadraticModuleCat`.
+
+-/
+
+open CategoryTheory
+
+universe v u
+
+variable (R : Type u) [CommRing R]
+
+/-- The category of `R`-bialgebras. -/
+structure BialgebraCat extends Bundled Ring.{v} where
+  [isBialgebra : Bialgebra R α]
```

Comment:
My first reaction to this was "this shouldn't be called `isSomething` because it's not a Prop", but I now see that this is also the style used in `AlgebraCat`. That doesn't mean I like it though :-/ I started a Zulip discussion [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/AlgebraCat.2EisRing.20isn't.20a.20Prop/near/443330331) and if we converge on e.g. `instBialgebra` then maybe you should make that change.

Reply:
Thanks! `instBialgebra` sounds better to me, have changed it